lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/DMLUtils.java

```diff
@@ -241,6 +264,39 @@ public static String createBatchDelete(TableMapping mapping){
 		addBackupIdInList(builder, mapping);
 		return builder.toString();
 	}
+
+	/**
+	 *	Build a 'select sum(crc32(etag|id)) statement for given mapping
+	 */
+	public static String createSelectSumCrc32ByIdRangeStatement(TableMapping mapping) {

```

Comment:
Change the CRC32 statement to combine the ID and ETAG:
concat(ID, '@', ifnull(ETAG, 'NULL')
Cannot assume ETAG column exists and ETAG not null or unique.